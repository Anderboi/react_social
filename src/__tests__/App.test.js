import { render, screen, fireEvent } from "@testing-library/react";
import SocialApp from "../App";

describe("MainApp component check", () => {
  test("renders learn react link", () => {
    render(<SocialApp />);
    const logoElement = screen.getByText(/minimal/i);
    const loginInput = screen.getByPlaceholderText(/email/i);
    expect(logoElement).toBeInTheDocument();
    expect(loginInput).toBeInTheDocument();
    expect(loginInput).toMatchSnapshot();
  });

  // test("Follow button toggle", () => {
  //   render(<SocialApp />);
  //   const button = screen.queryByTestId("profile-btn");
  //   expect(button).toBeInTheDocument();
    
  // });
});
