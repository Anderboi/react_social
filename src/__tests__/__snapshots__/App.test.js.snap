// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainApp component check renders learn react link 1`] = `
<input
  class="input"
  name="email"
  placeholder="Email"
  type="email"
/>
`;
